mockredis/client.py

```diff
@@ -1354,6 +1356,27 @@ def _normalize_command_response(self, command, response):
 
         return response
 
+    # Config Set/Get commands #
+
+    def config_set(self, name, value):
+        """
+        Set a configuration parameter.
+        """
+        self.redis_config[name] = value
```

Comment:
Also, we get `redis.exceptions.ResponseError: Unsupported CONFIG parameter: a` for an invalid config. We probably need a list of configs to mimic the ResponseError. Not required for this PR though.